versions.tf

```
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.14, < 2.0"
+  required_version = ">= 1.0, < 2.0"
```

we should force exclude broken versions.. ;)

```suggestion
  required_version = "~> 1.0, != 1.1.0, != 1.1.1"
```